models/emailer.js

```diff
@@ -46,6 +45,98 @@ exports.setup_potential_trade_email = function(user_id, owned_book, target_user,
 };
 
 
+exports.send_rejected_trade_email = function (trade_id, next) {
+    foundTrades.get_trade_by_id(trade_id, function(err, result) {
+        if (err) next(err);
+        for (var i = 0; i < result.length; i++) {
+            module.exports.setup_potential_trade_email(result[i].user_id, result[i].book_have, result[i].target_id, result[i].book_want, function(err, email_data) {
+                var name, email, wanted_book_name, wanted_book_author, have_book_name, have_book_author, target_name;
+                var replacements = {
+                    owner_user_name: email_data.user_name,
+                    target_user_name: email_data.target_name,
+                    target_book_name: email_data.wanted_book_name,
+                    target_book_author: email_data.wanted_book_author,
+                    owned_book_name: email_data.have_book_name,
+                    owned_book_author: email_data.have_book_author
+                }
+                var template = handlebars.compile(rejected_email_html);
+                var custom_html = template(replacements);
+
+                sgMail.setApiKey(process.env.EMAIL_KEY);
+                const msg = {
+                    to: email_data.user_email,
+                    from: process.env.EMAIL_ID,
+                    subject: 'Your Loop Textbook Trade Has Been Rejected',
+                    html: custom_html,
+                };
+                sgMail.send(msg);
+            });
+        }
+    });
+
+    next(true);
+};
+
+exports.send_accepted_trade_email = function (trade_id, next) {
+    foundTrades.get_trade_by_id(trade_id, function(err, result) {
+        if (err) next(err);
+        var emails = [];
+        var replacements = [];
+        logger.log(result);
+        var counter = 0;
+        for (var i = 0; i < result.length; i++) {
```

Comment:
style: Seems like you and jerry have different styles on space before curly braces or not